drivers/sensor/bme280/Kconfig

```diff
@@ -112,6 +112,19 @@ config BME280_FILTER_16
 	bool "16"
 endchoice
 
+choice
+	prompt "BME280 forced mode"
+	default BME280_FORCED_MODE_OFF
+	help
+	  Select whether measurements must be triggered upon request. In forced mode,
+	  after the measurement is finished, the sensor returns to sleep mode. Turning this
+	  on leads to less power consumption and less self-heating.
+config BME280_FORCED_MODE_OFF
+	bool "forced mode off"
+config BME280_FORCED_MODE_ON
+	bool "forced mode on"
+endchoice
```

Comment:
A plain `config` might be better, since it's just an on/off switch:

```
config BME280_FORCED_MODE
	bool "Only measure on request (forced mode)"
	help
	  If enabled, measurements must be ...
```

Reply:
(Symbols default to `n` by the way. See https://docs.zephyrproject.org/latest/guides/kconfig/index.html#redundant-defaults.)

Reply:
Agree

Reply:
Done